files/logrotate.conf

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
I think you can remove the license header from the *.conf files and add them to the ignore list

Reply:
It has been asked ona  previous comment by Greg to preserve the license headers in these files. https://github.com/canonical/pollen-operator/pull/2#discussion_r1223210656 if you both agree I can delete them.